ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/namespaceView.test.js

```diff
@@ -1,9 +1,10 @@
 import withAuth from '../../../helpers/basicAuth';
 
 import {
+    interactAndWaitForDeploymentList,
     visitWorkloadCveOverview,
     visitNamespaceView,
-    // waitForTableLoadCompleteIndicator,
+    // waitForTableLoadCompleteIndicator, // because of ROX-30492
```

Comment:
Is this saying the fix for ROX-30492 involves using this? Was a little confused by this. Can we add a "@TODO" so it's easier to find too?

Reply:
Aha, commented out because it caused the first false positive. I will push a commit to just delete it.